src/Admin/UserAdmin.php

```diff
@@ -152,6 +167,9 @@ protected function configureListFields(ListMapper $listMapper)
             ->addIdentifier('email')
             ->add('creationTime')
             ->add('updatedTime')
+            ->add('hasRecoverySecretBox', 'boolean', [
+                'label' => 'Recovery Token',
```

Comment:
I can imagine that this label causes the column to become very wide and take a lot of space? That would be unfortunate for a boolean column which has only 0/1 as values. Please ignore the comment if my assumption is wrong.

Reply:
I added a screenshot in the comments here. So you can get a feeling for the visual change.